assignment-01.md

```diff
@@ -5,26 +5,82 @@ E.g. Question <br>
+
+* References: [1](https://www.freshbooks.com/hub/reports/write-an-annual-report), [2](https://www.investopedia.com/articles/basics/10/efficiently-read-annual-report.asp), [3](https://corporatefinanceinstitute.com/resources/knowledge/finance/annual-report/)
+
 8. You have RM 10,000 savings and want to purchase shares as an investment for earning side income. 
 You have listed two potential companies and need to select one company to invest. 
 Comparing _Pentamaster Corporation Bhd._ and _ViTrox Corporation Bhd._ Justify your investment choice.
 
+![KLSE:PENTA Google Trends](img/penta-klse-gt.png "KLSE:PENTA Google Trends")
+![KLSE:VITROX Google Trends](img/vitrox-klse-gt.png "KLSE:VITROX Google Trends")
+
+* **Pentamaster** focuses on automation manufacturing and technology solutions provider while **Vitrox** focuses on design and manufacture of vision inspection equipment and embedded electronics devices. Both are **future-proof** in ways that they will not be phased out easily; and should it be phased out, they can rcontinue on with other similiar sectors.
+
+* In 5 year period:
+  * ViTrox's stocks increases from RM1.12 to RM7.25 up to **550%** increase in value.
+  * Pentamaster's stocks increases from RM0.48 to RM3.60 up to **650%** increase in value.
+
+* Because of the COVID-19 pandemic and the Malaysian political crisis, there are quite a few drops of stock value for both in the short term. In our case since investment is for long term it shouldn't be much of a problem.
+
+* Comparing both compnay using Annual Report in 2020:
+  * The earnings per share of ViTrox is **22.40sen** and **9.95sen** for Pentamaster.
+  * The cash ratio (cash/liabilities) for Vitrox is **1.67** and for Pentamaster is **3.69**. Pentamaster have more cash to cover their debts in the short term.
+  * The net income growth of ViTrox and Pentamaster is **32.60%** and **-14.64%** respectively. It should be noted that ViTrox's growth fell for 24.49% in the year 2019.
+  * The P/E ratio of ViTrox and Pentamaster is **65.42** and **76.79** respectively. Both of them are quite similiar in value.
+
+* As of 19/3/2022, the share price of ViTrox and Pentamaster is **RM7.25** and **RM3.60** respectively.
+
+* In conclusion, both of them have similiar values in terms of their business sectors, financial situation and more. But most notably Pentamaster's are traded at a cheaper valuation with better cash ratio and long term growth. I would choose **Pentamaster** as my primary stock but it's better to diversify (preferably other sectors) and invests in suitable proportions to minimize risks.
+
+* Referemces: [KLSE:PENTA - Google Trends](https://www.google.com/finance/quote/PENTA:KLSE), [KLSE:VITROX - Google Trends](https://www.google.com/finance/quote/VITROX:KLSE), [ValueInvestAsia](https://valueinvestasia.com/vitrox-corporation-berhad-vs-pentamaster-corporation-berhad/), [P/E 1](https://www.investopedia.com/terms/p/price-earningsratio.asp)
```

Comment:
Impressive attempt! Seems like you grasped the idea of financial analysis very well! Surely AKW104 won't be difficult for you! Well done.